src/app/fsat/fsat-report/input-summary/field-data-summary/field-data-summary.component.html

```diff
@@ -96,7 +96,7 @@ <h5>Field Data
               </td>
               <td *ngFor="let mod of fieldData.modifications" [ngClass]="{'indicate-report-field-different': checkDiff(fieldData.baseline.loadEstimatedMethod, mod.loadEstimatedMethod, 'loadEstimatedMethodDiff', 0)}">
                 <span *ngIf="mod.loadEstimatedMethod == 0">
```

Comment:
Should be &mdash; for either loadEstimatedMethod == 0 or loadEstimatedMethod == 1 so you can remove the `<span *ngIf`  and just put &mdash; &mdash;